package.json

```diff
@@ -14,7 +14,8 @@
   },
   "devDependencies": {
     "husky": "^8.0.3",
-    "lint-staged": "^13.1.2"
+    "lint-staged": "^13.1.2",
+    "pretty-quick": "^3.1.3"
   },
   "lint-staged": {
     "**/*.{js}": [
```

Comment:
insert  `prettier --write` before `eslint`, then remove `pretty-quick`

Reply:
keep pretty-quick to allow format code manually